src/apps/games/tests/test_game.py

```diff
@@ -134,7 +134,7 @@ def setup_method(self):
         self.bad_games.extend(self.create_games_with_defaults(board_size_x=45))
         self.bad_games.extend(self.create_games_with_defaults(board_size_y=2))
         self.bad_games.extend(self.create_games_with_defaults(board_size_y=45))
-        self.bad_games.extend(self.create_games_with_defaults(handicap=25))
+        self.bad_games.extend(self.create_games_with_defaults(handicap=2500))
```

Comment:
Ah yes !

Reply:
Note to myself, setup a CI to run the tests.

Reply:
Maybe we should test the view: eg https://github.com/pydanny/cookiecutter-django/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/%7B%7Bcookiecutter.project_slug%7D%7D/users/tests/test_views.py

Reply:
A CI to run the tests would be great!